# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/error-support-macros/.cargo-checksum.json
# Commit: 3326192e8db6
# Full Hash: 3326192e8db60772cb3d6a0f7987dc093c8ef35a
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-07 16:07:33
# Regressor Bug: 1818349
# File Overlap Count: 23
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Bug 1818349 Part 1: Vendor new version of application-services r=markh
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171181
# ==============================================================================

diff -r d7be17cdae33 -r 3326192e8db6 third_party/rust/error-support-macros/.cargo-checksum.json
--- a/third_party/rust/error-support-macros/.cargo-checksum.json	Tue Mar 07 06:59:50 2023 +0000
+++ b/third_party/rust/error-support-macros/.cargo-checksum.json	Tue Mar 07 07:42:14 2023 +0000
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"2a41a998c288fd049dd4dd62bc20e3ad13ded43642c27880dd5af0260bb72865","src/argument.rs":"1e4ec4140c536d2feea30eb1e4eef3a6e24533192108e91610ee5c2e75eb5e75","src/lib.rs":"fbd9c048ac86c16ef388fbb9a3c40f60d3aecab41a902740c81b3df6ee6631fe"},"package":null}
\ No newline at end of file
+{"files":{"Cargo.toml":"2a41a998c288fd049dd4dd62bc20e3ad13ded43642c27880dd5af0260bb72865","src/argument.rs":"bb97e801ce2c80b878328b15783678b913c2f34cf6f26a60d894c8da6b4e47aa","src/lib.rs":"8dd5b6225791730881a3500c3013c48678879430d859d0b92ac9dad4c42b04e0"},"package":null}
\ No newline at end of file